geopolars/src/geoseries.rs

```diff
@@ -104,6 +131,15 @@ pub trait GeoSeries {
 }
 
 impl GeoSeries for Series {
+    fn affine_transform(&self, matrix: impl Into<AffineTransform<f64>>) -> Result<Series> {
+        let transform: AffineTransform<f64> = matrix.into();
+        let output_vec: Vec<geo::Geometry> = iter_geom(self)
```

Comment:
Should we be consistent on using either `geo::Geometry` or bare `Geometry`? It looks like below you use mainly `Geometry`.

Reply:
yup changed this to use just Geometry